utils/build.ps1

```diff
@@ -1096,6 +1096,40 @@ function Get-Dependencies {
     Expand-Archive -Path $source -DestinationPath $destination -Force
   }
 
+  function Expand-TarBall {
+    param
+    (
+        [string]$TarBallFileName,
+        [string]$BinaryCache,
+        [string]$ExtractPath,
+        [bool]$CreateExtractPath = $true
```

Comment:
These map to names, so please rename these:
TarBallFileName -> Source
ExtractPath -> Destination
CreateExtractPath -> CreateDestination

Do we ever invoke this with `CreateDestinattion` being `$false`? If not, perhaps we should just drop the parameter.